main.py

```diff
@@ -274,9 +296,9 @@ async def command_list(message: Message):
 
 async def scrape_channel_media(
     channel: TextChannel,
-) -> List[Tuple[Member, str, str, str]]:
+) -> List[Tuple[Message, Attachment, str]]:
     # A list of (uploader, filename, local filepath, message jump url)
```

Comment:
This comment needs an update.

```suggestion
    # A list of (original message, message attachment, local filepath)
```